sdk/python/requirements.txt

```diff
@@ -1,19 +1,48 @@
-urllib3>=1.15,<1.25 #Fixing the version conflict with the "requests" package caused by the kubernetes package
-six>=1.10
-certifi
-python-dateutil
-PyYAML
-google-cloud-storage>=1.13.0
-kubernetes>=8.0.0, <=10.0.0
-PyJWT>=1.6.4
-cryptography>=2.4.2
-google-auth>=1.6.1
-requests_toolbelt>=0.8.0
-cloudpickle==1.1.1
-kfp-server-api==0.3.0
-argo-models == 2.2.1a
-jsonschema >= 3.0.1
-tabulate == 0.8.3
-click == 7.0
-Deprecated
-strip-hints
+#
+# This file is autogenerated by pip-compile
+# To update, run:
+#
+#    pip-compile --output-file=requirements.txt requirements.in
+#
+argo-models==2.2.1a       # via -r requirements.in
+attrs==19.3.0             # via jsonschema
+cachetools==4.0.0         # via google-auth
+certifi==2019.11.28       # via kfp-server-api, kubernetes, requests
+chardet==3.0.4            # via requests
+click==7.1.1              # via -r requirements.in
+cloudpickle==1.3.0        # via -r requirements.in
+deprecated==1.2.7         # via -r requirements.in
+google-api-core==1.16.0   # via google-cloud-core
+google-auth==1.11.3       # via -r requirements.in, google-api-core, google-cloud-storage, kubernetes
+google-cloud-core==1.3.0  # via google-cloud-storage
+google-cloud-storage==1.26.0  # via -r requirements.in
+google-resumable-media==0.5.0  # via google-cloud-storage
+googleapis-common-protos==1.51.0  # via google-api-core
+idna==2.9                 # via requests
+importlib-metadata==1.5.0  # via jsonschema
+jsonschema==3.2.0         # via -r requirements.in
+kfp-server-api==0.3.0     # via -r requirements.in
```

Comment:
Just curious, why we have `kfp-server-api==0.3.0` while in requirements.in we have `kfp-server-api==0.2.5`?

Reply:
sounds like we should drop kfp-server-api from requirements.in

Reply:
Sorry. It was a bug introduced during resolving merge conflict (someone has reformatted setup.py).
I'll set it to 0.3.0 in both places.